### ISSUE TITLE
Multiple styles

### ISSUE THREAD

**Issue by asok** [jwiegley/alert#53]
Hi,
I have a question - is there an easy way to have combine few styles together? What I mean is that, I would like for some messages to be logged via log4e and also via growl.

**Comment by jwiegley** [comment]
Yes, it should be simple to chain handlers, you just have to register multiple of them.

**Comment by pataquets** [comment]
I was looking for it in the docs and I almost thought it was not possible.
I could use some example on how to do it (pardon my ignorance). Anyone kind to provide any? It would be much appreciated.
